packages/form/src/form/utils/getRelationalObjectKey.ts

```diff
@@ -1,4 +1,9 @@
+import last from 'lodash/last'
+
 // Always returns the singular version of the key
-const getRelationalObjectKey = (key) => `${key.split('_id')[0]}`
+const getRelationalObjectKey = (key: string, keepPrefix: boolean = true) =>
```

Comment:
This is new... Should be shouldKeepPrefix = true

Reply:
By the way what prefix is this? Can help to leave a comment in the code. what prefix is and what is an example of it?

Reply:
By default we want to keep the key prefix
e.g for *LinesFieldArray: lines.[rowIndex].product_id -> lines.[rowIndex].product
But for searching/filtering: we only want the exact key
e.g lines.order_form_line.order_form.sales_order.project_id -> product